superset/dashboards/filter_sets/schemas.py

```diff
@@ -34,7 +34,7 @@ class JsonMetadataSchema(Schema):
 
 
 class FilterSetSchema(Schema):
-    json_metadata_schema: JsonMetadataSchema = JsonMetadataSchema()
+    json_metadata_schema: JsonMetadataSchema = JsonMetadataSchema(unknown=INCLUDE)
```

Comment:
We've hit these kinds of problems on the previous schema definitions before, and I wonder if we should just go with a raw schema for these JSON blobs for now? Having a defined schema but allowing for `unknown=INCLUDE` doesn't feel totally right - on one hand we're enforcing schema, but in practice not really. Thoughts?

Reply:
it enforces the required field and helps what should be inside but still, it is ok to let other fields who used only by the front.

Reply:
@dpgaspar we've discussed similar problems earlier on on the other endpoints - do you have any thoughts on using `unknown=INCLUDE` for these schemas? My fear is that this leads to gradual decay of the schema, as it won't raise errors on unknown fields, hence as the real schema evolves, this one stays stagnant. To quote the Finnish singer Aki Sirkesalo: "Annoin pikkusormen, se vei koko käden" (I gave my little finger, it took the whole hand).

Reply:
It depends on the context and the situation. 
right now the backend does nothing with that metadata so it's ok.
I don't understand why do you fear... if logic will be added, schema validation is probably required, and that is the reviewer's responsibility. if you don't trust others they will enforce it, this is different problem

Reply:
This is a general observation with regard to typing - where it isn't strictly enforced, over time it tends to erode. Since filter sets are currently fully restricted to the `apache/superset` repo (=no multirepo problems associated with sharing typing context between `superset-ui` and `apache/superset`), I don't see a major problem making sure that these schemas are in sync. See for instance the dashboard metadata payload schema which has not needed to be updated since active feature development of native filters stabilized: https://github.com/apache/superset/blob/5866d5ebb064648b6ac23c498cc252207f62dfd1/superset/dashboards/schemas.py#L108-L129

Expressing this differently, is there a reason why we can't just add the known extra fields to the schema in this PR, and do the same any follow-up PRs when we need to add new properties?

Reply:
@villebro 
Ok, you convinced me, so we don't need that PR and we will ensure no extra  fields will be sent 
I'm closing it